go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/mitchellh/mapstructure v1.1.2
 	github.com/vishvananda/netlink v1.0.0
 	github.com/vishvananda/netns v0.0.0-20190625233234-7109fa855b0f // indirect
+	github.com/yggdrasil-network/yggdrasil-extras v0.0.0-20201206140615-845aca5e7894 // indirect
```

Comment:
Can you `go mod tidy` to see if this goes away? I see no reason why `yggdrasil-extras` should be included here (and it introduces a circular dependency), so I would expect this to not be needed.

Reply:
I expect this has happened because I've done `./build -i` and it's been pulled in. `go mod tidy` has removed it.